core/src/cgroup.rs

```diff
@@ -173,12 +176,15 @@ impl CGroup {
         fs::write(killfile, "1")?;
 
         // Check if all processes were terminated successfully
-        for i in 1..=64 {
-            trace!("kill iteration ({i}/64)");
+        let start = Instant::now();
+        trace!("Killing with a 1s timeout");
```

Comment:
now it would make sense to also use the constant in these trace logs.

`trace!("Killing with a {KILLING_TIMEOUT:?} timeout")`